Documentation/configuration.md

```diff
@@ -133,7 +133,7 @@ To start etcd automatically using custom settings at startup in Linux, using a [
 The security flags help to [build a secure etcd cluster][security].
 
 ##### -ca-file [DEPRECATED]
-+ Path to the client server TLS CA file.
++ Path to the client server TLS CA file. The recommended equivalent alternative is "--trusted-ca-file '' --client-cert-auth".
```

Comment:
@robszumski If user uses '-ca-file ca.crt' before, he could change to '-trusted-ca-file ca.crt --client-cert-auth' and etcd will perform the same. I want to provide a practical way for user to stop using deprecated flags.